src/runtime/classderived.cs

```diff
@@ -786,7 +786,7 @@ public static void InvokeCtor(IPythonDerivedType obj, string origCtorName, Objec
                         PyObject[] pyargs = new PyObject[args.Length];
                         for (int i = 0; i < args.Length; ++i)
                         {
-                            pyargs[i] = new PyObject(Converter.ToPython(args[i], args[i].GetType()));
+                            pyargs[i] = new PyObject(Converter.ToPython(args[i], args[i]?.GetType()));
```

Comment:
I think we are not ready for C# 6+ only support. Please do not use "Null-conditional operators" in C# 4.0 codebase.